mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -223,33 +264,32 @@ def _signed_url_upload_file(self, credentials, local_file):
         except Exception as err:
             raise MlflowException(err)
 
-    def _upload_to_cloud(self, cloud_credentials, local_file, artifact_path):
-        if cloud_credentials.credentials.type == ArtifactCredentialType.AZURE_SAS_URI:
-            self._azure_upload_file(cloud_credentials.credentials, local_file, artifact_path)
-        elif cloud_credentials.credentials.type in [
+    def _upload_to_cloud(
+        self, cloud_credential_info, src_file_path, dst_run_relative_artifact_path
```

Comment:
Tweaked some parameter names to be more descriptive and consistent with proto field naming (e.g. `cloud_credentials -> cloud_credential_info` to indicate a single entry within the `credential_infos` proto response field)